### ISSUE TITLE
Hide accessibility statement in Epub file

### ISSUE THREAD

**Issue by juliusross1** [juliusross1/Accessible-LaTeX-Thesis-Template#1]
There should be a way to have latexml ignore the accessibility statement automatically.  Might need to think about how this would affect pagination
